addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -629,7 +630,7 @@ export class AnalyticDistribution extends Component {
 
     onWindowResized() {
         // popup ui is ugly when window is resized, so close it
-        if (this.isDropdownOpen) {
+        if (this.isDropdownOpen && isMobileOS()) {
```

Comment:
I don't understand. Shouldn't this be if it's _not_ in mobile?

```
if resize:
	if dropdown_open:
		if mobile:
			don't close (because we resize often in mobile, even when spawning keyboard to type)
		else:
			close
```

Reply:
My bad, I just changed it :expressionless: